site/cypress/e2e/docker_ui.spec.js

```diff
@@ -0,0 +1,154 @@
+const docker_ui_path = '/admin/docker';
+
+const waitAndReloadUntil = (condition, timeout, wait = 100) => {
+    cy.wait(wait);
+    cy.reload();
+    cy.then(() => {
+        return condition().then((result) => {
+            if (result || timeout <= 0) {
+                return result;
+            }
+            return waitAndReloadUntil(condition, timeout - wait, wait);
+        });
+    });
+};
```

Comment:
This function might be useful elsewhere as well.  Perhaps it could be moved to the libraries instead of residing in this file.

Reply:
Sure, how does registering a command in `/site/cypress/support/commands.js` sound?

Reply:
That seems like a good idea.  I know of at least a few other use cases for this function.  In particular, I was struggling with this exact issue some time ago when I was working on Lichen UI tests.